app/script/entity/Conversation.js

```diff
@@ -398,15 +398,15 @@ z.entity.Conversation = class Conversation {
     return new Date(timestamp).toISOString();
   }
 
-  getNumberOfBots() {
-    return this.participating_user_ets().filter(userEntity => userEntity.isBot).length;
+  getNumberOfServices() {
+    return this.participating_user_ets().filter(userEntity => userEntity.isService).length;
   }
 
-  getNumberOfParticipants(countSelf = true, countBots = true) {
+  getNumberOfParticipants(countSelf = true, countServices = true) {
     const adjustCountForSelf = countSelf && !this.removed_from_conversation() ? 1 : 0;
 
-    if (!countBots) {
-      const numberOfParticipants = this.participating_user_ets().filter(userEntity => !userEntity.isBot).length;
+    if (!countServices) {
```

Comment:
How about we simplify the body of this method to consume the `getNumberOfServices`?
something like `totalNumber - numberOfServices` if the `countServices` is true else just `totalNumber`